main/src/ca/uwaterloo/flix/language/phase/Resolver.scala

```diff
@@ -1662,6 +1662,13 @@ object Resolver {
             case (e1, e2) => ResolvedAst.Expression.FixpointProject(pred, e1, e2, loc)
           }
 
+        case NamedAst.Expression.Instanceof(exp, className, loc) =>
+          val eVal = visitExp(exp, env0)
+          val clazzVal = lookupJvmClass(className, loc);
```

Comment:
Tests that fail this check would go in `TestResolver`, e.g. if you want a test that does `foo instanceof DoesNotExist`.

Reply:
I missed this, I will add some Resolver tests

Reply:
I've added a Resolver test, but also a couple of Parser tests. I think there is really only one Resolver error - "does not exist", and the parser will pick up ill-formed names.